### PR TITLE
refactor startup

### PR DESCRIPTION
**Pull request by recallsong**
#### What type of PR is this?

/kind refactor

#### What this PR does / why we need it:
refactor startup

#### Specified Reivewers:
/assgin @johnlanni @Effet @luobily

### CODE REVIEW

File: modules/volume-provisioner/provider.go [hunk 1]
```diff
@@ -17,41 +17,10 @@ package volumeprovisioner
 import (
 	"context"
 
-	"github.com/sirupsen/logrus"
-
 	"github.com/erda-project/erda-infra/base/servicehub"
+	"github.com/sirupsen/logrus"
```
Comment (luobily) [inline modules/volume-provisioner/provider.go:21]:
```suggestion
	"context"

	"github.com/sirupsen/logrus"

	"github.com/erda-project/erda-infra/base/servicehub"
```

Comment (luobily) [reply modules/volume-provisioner/provider.go:21]: the order of imports:
- Standard library imports
- Third party imports,
- Own imports.

**Review by luobily — CHANGES_REQUESTED** [review]

File: modules/volume-provisioner/provider.go (hunk 1 above)
Comment (recallsong) [reply modules/volume-provisioner/provider.go:21]: the order of imports:
1、Standard package imports
2、Other imports
it is enough, and not necessary to distinguish between Third party imports and Own imports.

**Review by liuhaoyang — APPROVED** [review]

**Review by luobily — APPROVED** [review]
lgtm

**Pull request merged**